operator/grafana/controller-resources-metrics-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: controller-resources-metrics-dashboard
+  namespace: kcp-system
```

Comment:
shouldnt these be in the default namespace and then be automatically corrected by the kustomization? I am concerned why we put these outside of the kustomization

Reply:
make sense, will change it 👍